.evergreen/orchestration/configs/replica_sets/replicaset-old.json

```diff
@@ -12,10 +12,9 @@
                 "bind_ip": "::,0.0.0.0"
             },
             "rsParams": {
-                "priority": 99,
                 "tags": {
                     "ordinal": "one",
-                    "dc": "pa"
+                    "dc": "ny"
                 }
             },
             "server_id": "RS-OLD-one"
```

Comment:
Any reason not to change the `server_id` values as well?

Reply:
Must've missed this. I noticed the configs provided by drivers-evergreen-tools don't include this, is there any reason to keep these around?

Reply:
This is only useful if we want to manage servers in a replica set directly (see https://github.com/10gen/mongo-orchestration/pull/168). None of our tests rely on that, so I think we can remove this. MO will generate a UUID if this field is omitted (see [here](https://github.com/10gen/mongo-orchestration/blob/066ec58c817947ca136c25690a705fd4e4db2a7b/mongo_orchestration/servers.py#L512)).

For historical context, this was likely related to https://github.com/estolfo/mo-tests.

Reply:
Removed the `server_id` config from this file.